tables/latex.txt

```diff
@@ -198,8 +198,8 @@ _x	ₓ	0
 ^(	⁽	0
 ^)	⁾	0
 ^n	ⁿ	0
-###
-### Unicode symbols interval 2100-214F
+### 
```

Comment:
I believe there is an unintentional trailing space here.

Reply:
OK, I’ll fix the trailing space.